lib/fluent/plugin/auditd.rb

```diff
@@ -1,120 +1,113 @@
+require 'time'
+
 module Fluent
   class Auditd
     class AuditdParserException < StandardError
     end
-    
-    MSG = 'msg'
-    TIMESTAMP = 'time'
-    AUDIT = 'audit'
 
+    IN_HOST_PID = 'pid'
+    IN_HOST_UID = 'uid'
+    IN_HOST_AUID = 'auid'
+    IN_HOST_SESSION = 'ses'
+    IN_HOST_SELINUX_LABEL = 'subj'
+    IN_HOST_HOSTNAME = 'hostname'
+    IN_VM_AUID = 'auid'
+    IN_VM_HOSTNAME = 'hostname'
+    IN_VM_IMAGE = 'vm'
+    IN_VM_PID = 'vm-pid'
+    IN_VM_USER = 'user'
+    IN_VM_EXE = 'exe'
+    IN_VM_REASPON = 'reason'
+    IN_VM_OPERATION = 'op'
+    IN_VM_RESULT = 'res'
+    IN_EVENT_TYPE = 'virt_control'
+
+    OUT_HOST_PID = 'systemd.t.PID'
```

Comment:
This gets written to `record['systemd']['t']['PID']`

And you'll want to make sure that `record['systemd']` exists, and create it if not, make sure `record['systemd']['t']` exists and create it if not
```
   ((record['systemd'] ||= {})['t'] || = {})['PID'] = pidvalue
   # the first line ensures that record['systemd']['t'] exists
   record['systemd']['t']['AUDIT_LOGINUID'] = auidvalue
   ...
```